src/components/NavBar.tsx

```diff
@@ -56,7 +59,7 @@ const NavBar: React.FC = () => {
         />
         <div
           className={classNames(
-            "fixed top-0 left-0 h-full bg-gray-darkest transition-[max-width] ease-in-out duration-500",
+            "fixed top-0 left-0 h-full bg-gray-darkest transition-all ease-in-out duration-500 mt-12",
```

Comment:
Is there a reason this doesn't work with `w-full` always and toggling `max-width`?